src/api/apiCore.cpp

```diff
@@ -355,6 +364,15 @@ void TapeCore::tape_entity_set_intensity_texture(const YAML::Node& yamlNode, Pla
 	                                 state.textures.at(yamlNode[1].as<TapeAPIObjectID>()));
 }
 
+rgl_status_t rgl_is_entity_valid(rgl_entity_t entity, bool* out_valid)
+{
+	auto status = rglSafeCall([&]() {
+		*out_valid = Entity::instances.find(entity) != Entity::instances.end();
+	});
+	TAPE_HOOK();
+	return status;
```

Comment:
```suggestion
	auto status = rglSafeCall([&]() {
		CHECK_ARG(out_valid != nullptr);
		*out_valid = Entity::instances.contains(entity);
	});
	TAPE_HOOK(entity, out_valid);
	return status;
```

Reply:
Thank you. This has been fixed, see comment above.